setup.py

```diff
@@ -33,7 +33,7 @@ def load_requirements(*requirements_paths):
 
 setup(
     name='edx-submissions',
-    version='1.2.0',
+    version='2.0.0',
```

Comment:
> I'm not sure how strict we are about using that.  I hear people say we use it, but we have a lot of code that just hangs out in v1.

You're adding more strictness to edx-submissions' "let's follow semver guidelines" process with this PR than has ever existed in this repo previously :rofl:.  I had been thinking that we could have added this in a backwards compatible way, but edx-ora2 is the only consumer so let's make it work exactly how we want.